app/controllers/users_controller.rb

```diff
@@ -44,12 +57,23 @@ def add_contact
     redirect_to root_path
   end
 
+  # rubocop:disable Metrics/AbcSize
```

Comment:
I would prefer changing code not the config of rubocop. Why should we use it at all if we are turning it off when it detects opportunities for improvements?

Reply:
In my opinion, sometimes methods will be more complex than 10 lines of code. But I am programming quite a lot of c++ lately so maybe I just have a different subjective view on this matter. Since I replaced this method with the one of  Scrumbledore's Army I don't need to disable rubocop here.